### PR TITLE
[AVFoundation] Add TV OS missing apis.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Comment by spouliot** [comment]
There's one tvOS introspection failure
> [FAIL] AVKit.AVContentProposalViewController should re-expose UIViewController::.ctor(String, NSBundle)
> [FAIL] iOSApiCtorInitTest.ApiCtorInitTest.DesignatedInitializer : 1 potential errors found in 318 designated initializer validated

File: src/avkit.cs [hunk 1]
```diff
@@ -215,6 +231,30 @@ interface AVPlayerViewControllerDelegate
 		[TV (9,0)]
 		[Export ("playerViewController:didSelectExternalSubtitleOptionLanguage:")]
 		void DidSelectExternalSubtitleOptionLanguage (AVPlayerViewController playerViewController, string language);
+
+		[NoiOS, TV (10,0), NoWatch, NoMac]
+		[Export ("playerViewController:timeToSeekAfterUserNavigatedFromTime:toTime:")]
+		CMTime TimeToSeekAfterUserNavigatedFromTime (AVPlayerViewController playerViewController, CMTime oldTime, CMTime targetTime);
```
Comment (spouliot) [inline src/avkit.cs]: remove `FromTime` suffix, it maps to the first parameter

File: src/avkit.cs [hunk 2]
```diff
@@ -215,6 +231,30 @@ interface AVPlayerViewControllerDelegate
 		[TV (9,0)]
 		[Export ("playerViewController:didSelectExternalSubtitleOptionLanguage:")]
 		void DidSelectExternalSubtitleOptionLanguage (AVPlayerViewController playerViewController, string language);
+
+		[NoiOS, TV (10,0), NoWatch, NoMac]
+		[Export ("playerViewController:timeToSeekAfterUserNavigatedFromTime:toTime:")]
+		CMTime TimeToSeekAfterUserNavigated (AVPlayerViewController playerViewController, CMTime oldTime, CMTime targetTime);
```
Comment (dalexsoto) [inline src/avkit.cs]: Get prefix

**Comment by dalexsoto** [comment]
LGTM 👍

File: src/avkit.cs [hunk 3]
```diff
@@ -95,7 +95,7 @@ interface AVPictureInPictureControllerDelegate
 	}
 	
 
-	[iOS (8,0)]
+	[iOS (8,0), TV (9,0)]
```
Comment (rolfbjarne) [inline src/avkit.cs]: tvOS 9? Shouldn't that be tvOS 10?

Comment (mandel-macaque) [inline src/avkit.cs]: We do have the class under /Applications/Xcode73.app/Contents/Developer/Platforms/AppleTVOS.platform//Developer/SDKs/AppleTVOS9.2.sdk/System/Library/Frameworks/AVKit.framework/Headers/AVPlayerViewController.h with a NS_CLASS_AVAILABLE_IOS (9_0)

**Comment by rolfbjarne** [comment]
LGTM 👍

**Pull request merged**